packages/taiko-client/cmd/flags/preconfapi.go

```diff
@@ -13,6 +15,27 @@ var (
 		Value:    9871,
 		EnvVars:  []string{"PRECONFAPI_PORT"},
 	}
+	PollingInterval = &cli.DurationFlag{
+		Name:     "preconfapi.pollingInterval",
+		Usage:    "Interval at which to poll",
+		Category: preconfAPICategory,
+		Value:    1 * time.Second,
+		EnvVars:  []string{"POLLING_INTERVAL"},
+	}
+	DBPath = &cli.StringFlag{
```

Comment:
Is this change to the DB just for testing or does it need to be merged into the main network?

Reply:
all of this so far is just for testing, this is currently just using. badger DB for an in memory / file based DB to index pre confirmed transactions and expose them via an API for the explorer to query, via polling. If we decide to use this approach in the future for real, it will be refactoref to accept a more complicated and robust database as an option via an interface instead of hard coded badgerdb in the indexer. If it's not needed for mainnet/etc, it will be removed.

Reply:
ok